packages/init/src/steps/install-lib.ts

```diff
@@ -103,7 +102,7 @@ export async function installLib(
       `Slice library "${libGithubPath}" was installed successfully`
     );
 
-    tracker?.Track.downloadLibrary(libGithubPath);
+    Tracker.get().trackDownloadLibrary(libGithubPath);
```

Comment:
should this be catch to not make the rest fails or does the fact that there is no await covers it ?